test/fixed_vector_test.cpp

```diff
@@ -369,6 +369,17 @@ TEST(FixedVector, Initializer)
     EXPECT_TRUE(are_equal(v2, std::array{66, 55}));
 }
 
+TEST(FixedVector, SizeDeduction)
+{
+    constexpr auto v1 = make_fixed_vector<int>({10, 11, 12, 13, 14});
+    static_assert(v1.size() == 5);
```

Comment:
Can you add another check for max_size() to show that it is not possible to add more entries?

https://en.cppreference.com/w/cpp/container/vector/max_size

Reply:
Done!